Core/src/main/java/com/plotsquared/core/util/RegionManager.java

```diff
@@ -167,9 +187,17 @@ public boolean copyRegion(final Location pos1, final Location pos2, final Locati
         final BasicQueueCoordinator copyTo = (BasicQueueCoordinator) blockQueue.getNewQueue(newWorld);
         copyFromTo(pos1, pos2, relX, relZ, oldWorld, copyFrom, copyTo, false);
         copyFrom.setCompleteTask(copyTo::enqueue);
+        if (actor != null && Settings.QUEUE.NOTIFY_PROGRESS) {
+            copyFrom.addProgressSubscriber(subscriberFactory.create(actor, taskManager, Settings.QUEUE.NOTIFY_INTERVAL, Settings.QUEUE.NOTIFY_WAIT,
+                StaticCaption.of("<prefix><gray>Current copy progress: </gray><gold><progress></gold><gray>%</gray>")));
```

Comment:
Why is this a static caption? This could be moved into a TranslatableCaption and append `<progress>%` as template.